angular/src/lib/pagination/pagination-arrow.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { PaginationArrowComponent } from './pagination-arrow.component';
+
+@NgModule({
+  declarations: [PaginationArrowComponent],
+  imports: [
+    CommonModule
+  ],
+  exports: [PaginationArrowComponent],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA]
+})
+export class PaginationArrowModule { }
```

Comment:
Need to add forRoot to new components, here's an example; it should be in all the component modules

export class TopbarModule {
  static forRoot(): ModuleWithProviders {
    return {
      ngModule: TopbarModule,
      providers: [],
    };
  }
}

Reply:
Thanks for you suggestion